README.md

```diff
@@ -117,12 +117,11 @@ The two most interesting/relevant methods are (simplified versions shown here):
 ```javascript
 // generate the JWT
 function generateToken(req){
-  var token = jwt.sign({
+  return jwt.sign({
```

Comment:
this was originally done because JSLint did not like returning an unassigned variable. 🙄 
I'm happy to merge this PR as I feel that this code is cleaner.
But for the record: these two are 100% equivalent from the V8 JIT's perspective.

Reply:
Sweet thanks for the reply!  Very cool project :)